src/environments/environment.tests.ts

```diff
@@ -4,5 +4,6 @@ export const environment = {
   ...common,
   production: false,
   apiEndpoint: 'https://zonemaster.net/api',
-  mock: true
+  mock: true,
+  configUrl: 'assets/app.config.tests.json'
```

Comment:
I cannot see that the file 'assets/app.config.tests.json' exists. Should it be 'assets/app.config.sample.json'? Or is the other file created some way?

Reply:
I use a non existing file on purpose, so that the config used is always the default (`environment.tests.ts`). I can document that in this file if needed.

Reply:
Yes, please do. Or maybe rename the file to 'assets/app.config.non-existent.json' to make it more obvious that it should not exist.

Reply:
Done